### PR TITLE
fixes build issue by supplying Accumulo version

### PR DESCRIPTION
**Pull request by keith-turner**
Tried to run bin/build against accumulo 2.1.1-SNAPSHOT and it failed looking for 2.1.0-SNAPSHOT resources.  This was because script was not passing the runtime Accumulo version.  Modified the script to pass the runtime version.

### CODE REVIEW

**Comment by Manno15** [comment]
I fixed this in #256

**Review by Manno15 — APPROVED** [review]

**Comment by Manno15** [comment]
The QA check should be fixed with the changes in #256

**Comment by keith-turner** [comment]
I think #256 would have hidden this problem, so I got lucky in that I attempted to use this repo before #256 was merged.  The build script was not always using what was maven installed locally.  Only uncovered this problem because 2.1.0-SNAPSHOT is no longer in the apache snapshot repo.

**Comment by keith-turner** [comment]
I'll merge this after #256, then maybe we can see if this one build.

**Comment by Manno15** [comment]
I re-ran the build.

**Pull request merged**
